chef_master/source/deprecations_locale_lc_all.rst

```diff
@@ -0,0 +1,47 @@
+=======================================================================
+Deprecation: Deprecation of lc_all from locale resource (CHEF-27)
+=======================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_locale_lc_all.rst>`__
+
+.. tag deprecations_locale_lc_all
+
+Setting *LC_ALL* variable is NOT recommended and can break Chef's parsing of command output in unexpected ways. Use one of the more specific *LC_* properties as needed.
+
+.. end_tag
+
+This deprecation warning was added after Chef 15.0.185. Support for property `lc_all` will likely be removed by Chef 15.1.0.
+
+
+Remediation
+=======================================================================
+
+Though it is inadvisable, still if there is a need to set *LC_ALL*, you may set this environment variable manually into any one of the files listed below.
+
+.. code-block:: ruby
+
+	export LC_ALL="<locale_name>"
+
+1. /etc/default/locale
```

Comment:
I would add this:

> Setting *LC_* variables varies by platform, but common locations to configure *LC_* variables are:

And then drop the note on "priority order" below.

Reply:
Updated